### ISSUE TITLE
Important Connection Issues and Recommendation to Address it

### ISSUE THREAD

**Issue by DJSixHub** [ValveSoftware/Dota2-Gameplay#12170]
### Description

Excuse my English, it is not my first language. My account has around 300 games. In most of those games i've played normally but recently, after the new client update or after some little updates that were done after the new client, i've had around 1100 PING on my games, and it cannot be tested before starting, because in region selection it appears with 140-170 PING, which is what i always had before the issues started. So, i enter my games, and after a few minutes, and recently specially after the 0:00 time, when the day starts in the game, my goes straight to 1000. It is impossible to play that way. I live in North America and play in USA EAST, but in happens on every region.

I know u guys cannot determine accurately when someone disconnects on purpose from a game, and ruining everyone else's game should be punished, but this is insane. U guys can know when someone is attempting to reconnect to a game many times, and even more, you have access to the quality of the connection during the game. U absolutely CAN tell when someone is disconnecting due to this issues, so my proposal is to change the way you punish those issues. I thing sending someone with a perfect behaviour score to low priority over what is a totally evident connection issue is not fair. I cannot even play with my friends because now my score is very low and im stuck in low priority forever.

PLS FIX WHATEVER U DID TO THE CONNECTION AND FIX THE PUNISHMENT FOR WHAT's a LEGITIMATE REASON TO NOT BE ABLE TO PLAY OR CONNECT. U HAVE THAT INFORMATION

**Comment by TobyFlendersonFromHR** [comment]
see ya in low prio, buddy
